Source/Dafny/DafnyOptions.cs

```diff
@@ -1063,19 +945,87 @@ not sound
     experimentalPredicateAlwaysGhost - Compiled functions are written `function`.
         Ghost functions are written `ghost function`. Predicates are always ghost
         and are written `predicate`.
+/disableScopes
+    Treat all export sets as 'export reveal *'. i.e. don't hide function bodies
+    or type definitions during translation.
+/allowGlobals Allow the implicit class '_default' to contain fields, instance functions,
+    and instance methods.  These class members are declared at the module scope,
+    outside of explicit classes.  This command-line option is provided to simplify
+    a transition from the behavior in the language prior to version 1.9.3, from
+    which point onward all functions and methods declared at the module scope are
+    implicitly static and fields declarations are not allowed at the module scope.
 
+---- Warning selection -----------------------------------------------------
+
+/warnShadowing  Emits a warning if the name of a declared variable caused another variable
+    to be shadowed
 /deprecation:<n>
     0 - don't give any warnings about deprecated features
     1 (default) - show warnings about deprecated features
     2 - also point out where there's new simpler syntax
+/warningsAsErrors
+    Treat warnings as errors.
+
+---- Verification options -------------------------------------------------
+
+/dafnyVerify:<n>
+    0 - stop after typechecking
```

Comment:
```suggestion
    0 - stop after resolution
```
I'm not sure of this one, but we mostly talk about "resolution" rather than "typechecking" when speaking of the phase after parsing.
I know you are just refactoring, but I thought maybe we could just update it based on usage.